RoboJackets/robocup-software
# :sparkles: Create Hypothetical Planning Services for Robots :sparkles:

## Description
In gameplay, it may be important for robots to plan a hypothetical path.  In these situations, it is likely the gameplay system will want to know how long it would take the robot to execute a given path, which is exactly what this pr enables.

## Associated / Resolved Issue
[allow for hypothetically planned trajectories](https://app.clickup.com/t/3347pp6)

## Steps to Test
I created a testing node in soccer/src/soccer/testing, which will execute a robot intent and print (in blue) the time estimate it will take to execute that path.  Literally just run the following commands
- make prf
- make run-sim | grep "testing_node"

**Expected result:** blue text should be printed to the screen displaying a time.

## Review Checklist


## NOTES:
Once this gets reviewed I'll remove the testing code, but its hard to demonstrate that something exists without showing that it works so it will remain for reviews.
